package.json

```diff
@@ -142,7 +144,11 @@
       "**/tests/ui/**/*.js",
       "**/tests/conf/**/*.js",
       "**/tests/answers-umd.js"
-    ]
+    ],
+    "transform": {
+      "^.+\\.js$": "babel-jest",
+      ".+\\.(svg)$": "jest-transform-stub"
```

Comment:
just asking for myself but why does only the babel-jest regex need a caret? And do we need parenthesis around the "svg" part?

Reply:
I was just following the example from the jest-transform-stub repo, but we don't need the caret so I'll remove it. We don't need the parenthesis either, but we can add those back if we ever want to add more types of assets like this: `".+\\.(css|styl|less|sass|scss|png|jpg|ttf|woff|woff2)$"`